nodedb.go

```diff
@@ -361,7 +363,7 @@ func (ndb *nodeDB) deleteVersion(version int64, checkLatestVersion, memOnly bool
 	return nil
 }
 
-// Saves orphaned nodes to disk under a special prefix.
```

Comment:
No, this also stores orphans for recentDB, which are in memory only. snapshotDB and recentDB have orphaning on different schedules, because deleting a version in memory (e.g. when pruning based on `KeepRecent`) should not cause it to be deleted on disk.

Reply:
But there is a call to `saveOrphan `, which uses `snapshotOrphanBatch`.

Reply:
Nvm, ignore that comment.